apps/server/src/modules/tool/controller/mapper/school-external-tool-request.mapper.ts

```diff
@@ -7,6 +7,7 @@ import { SchoolExternalTool } from '../../uc/dto/school-external-tool.types';
 export class SchoolExternalToolRequestMapper {
 	mapSchoolExternalToolRequest(request: SchoolExternalToolPostParams): SchoolExternalTool {
 		return {
```

Comment:
I think you should say `new SchoolExternalToolDO` here explicitly. (Not part of your PR but related: Why did you alias the type?)

Reply:
We don't wanna create a new object here